tests/pipelines/employees.py

```diff
@@ -92,10 +147,34 @@ class Transformer(Phase):
     steps = [
         combine_full_name,
         calculate_annual_salary,
-        calculate_bonus_percent
+        calculate_bonus_percent,
+        index_departments,
+        add_department_id
+    ]
+    extra_sources = [
+        'departments'
     ]
 
+class ManagerPhase(Phase):
```

Comment:
Just in order to align with and show an example of some of the common terminology, and assuming this example stays in our front page Readme, I wonder if we should do the phases as:
Enrichment: phase that indexes departments and merges in department id
Transformer: phase that does calculations and reorganizing managers, takes no extra sources